AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/core/CustomisedGsonBuilder.java

```diff
@@ -1,12 +1,10 @@
 package org.softwire.training.api.core;
 
-import com.google.gson.Gson;
-import com.google.gson.GsonBuilder;
-import com.google.gson.JsonSyntaxException;
-import com.google.gson.TypeAdapter;
+import com.google.gson.*;
```

Comment:
Personally I prefer importing everything explicitly here rather than using the * notation, but not something I feel strongly about and happy for you to have chosen this.

Reply:
I find it weird that it did this as I did not change any of the imports at any point, I don't know why it did this :o